### PR TITLE
support skaffold as a development env on Mac OS

### PR DESCRIPTION
**Pull request by steved**
Due to underlying library limitations, Forge doesn't support local development. Skaffold allows us to do the iteration cycle in a local minikube cluster based on Virtualbox.

Usage:
```
$ e2e/dependencies.sh
$ skaffold dev
[dev - run iteration loop]

$ skaffold debug --port-forward
[connect using local IDE / delve]
```

Better documentation _WIP_ before merging.

### CODE REVIEW

File: controllers/containerimagebuild_controller.go
Comment (sonnysideup) [inline controllers/containerimagebuild_controller.go:121]: I guess it makes sense to set the completed at datetime after a failure. Good catch.

**Review by sonnysideup — APPROVED** [review]
Looks good. I would just add a quick "Development" section to the README.

**Comment by steved** [comment]
@sonnysideup might be worth a second pass. Might also have to wait for #42 before I merge it.

**Review by saahildhulla — APPROVED** [review]

**Pull request closed without merging**
